src/main.cc

```diff
@@ -464,25 +416,20 @@ int main(void) {
             case (0x2):
                 /* BOOST mode 0x02 */
                 // when the rocket is in flight off launch
-                HAL_GPIO_WritePin(LED_STANDBY_GPIO_Port,  LED_STANDBY_Pin,  GPIO_PIN_RESET);
-                HAL_GPIO_WritePin(LED_ARMED_GPIO_Port,    LED_ARMED_Pin,    GPIO_PIN_SET);
+                HAL_GPIO_WritePin(LED_STANDBY_GPIO_Port, LED_STANDBY_Pin, GPIO_PIN_RESET);
+                HAL_GPIO_WritePin(LED_ARMED_GPIO_Port, LED_ARMED_Pin, GPIO_PIN_SET);
 
                 // condition to find when to deploy drogue parachute
-                if(afsData.altitude != -1)
-                {
-                    if(altData.altitude < prevAltitude)
-                    {
+                if (afsData.altitude != -1 && altData.altitude != prevAltitude) {
+                    if (altData.altitude < prevAltitude) {
                         tick++;
                         // 100 is found based on HZ of loop being around 50 HZ ish
-                        if(tick > 100)
-                        {
+                        if (tick > 5) {
```

Comment:
How fast/slow do we poll the altimeter for us to be confident the metric "five decreasing readings in a row" will never happen because of random noise?